app/services/csv_exporter.rb

```diff
@@ -90,7 +91,7 @@ class CSVExporter
     'in_target',
     'number_of_days_taken',
     'number_of_days_late',
-    'number_of_days_taken_after_extension',
+    'number_of_days_taken_after_extension'
```

Comment:
Not sure about this line, did you mean a new field name?

Reply:
No I had a new field name here and then realised that it wasn't needed - I removed the comma as it is not needed in any case 👍